pkg/rpc/server.go

```diff
@@ -209,6 +208,31 @@ Methods:
 			results = "Invalid assetid"
 		}
 
+	case "getaccountstate":
+		var err error
+
+		param, exists := reqParams.ValueAtAndType(0, "string")
+		if !exists {
+			err = fmt.Errorf("Please provide a valid string account address parameter")
```

Comment:
> Error strings should not start with a capital letter because they'll often be prefixed before printing:

https://github.com/golang/go/wiki/Errors

and, just use https://github.com/pkg/errors

Reply:
okay, done.